src/strgen/strgen.cpp

```diff
@@ -46,7 +46,11 @@ void CDECL strgen_warning(const char *s, ...)
 	va_start(va, s);
 	vseprintf(buf, lastof(buf), s, va);
 	va_end(va);
-	fprintf(stderr, LINE_NUM_FMT("warning"), _file, _cur_line, buf);
+	if (_show_todo > 0) {
+		fprintf(stderr, LINE_NUM_FMT("warning"), _file, _cur_line, buf);
+	} else {
+		fprintf(stderr, LINE_NUM_FMT("info"), _file, _cur_line, buf);
+	}
```

Comment:
why would the classification change depending on a setting? I am not sure I follow that logic. For who will it become a warning that doesn't think info is what he is looking for?

Reply:
I'm not sure to fully understand your question.
`-t` sets bit 0 of `_show_todo`, `-w` sets bit 1, and now english.txt sets bit 2.
So if any bit is set it's a warning, else it's an info.

Reply:
No, I am wondering why this if-statement exists in the first place. Why would it be a "warning" in some cases, and "info" in others. That always feels a bit weird, and often tells there is something else being a bit off. I haven't checked how other places do it. I just don't get why the log-level would change depending on `-t` for example.

Reply:
Okay, a bit closer look, I do not think this is the right approach. There are plenty of warnings that really are warnings, and never infos. Like invalid UTF-8, for example. What if we just add a new command like `strgen_translation_warning`, in that case I could understand this if-statement a lot more :)

Reply:
Nevermind all this, it is already messy as fuck. So it is not fair to ask from you to fix "everything" just to fix a bug :P So lets just do this.